src/app.module.ts

```diff
@@ -121,6 +124,16 @@ export class AppModule implements NestModule {
           // return 500 for files that do not exist instead of a 404
           exclude: ['/(.*)'],
         }),
+        TypeOrmModule.forRootAsync({
+          imports: [ConfigModule],
+          useFactory: (configService: ConfigService) => {
+            return {
+              ...{ autoLoadEntities: true, manualInitialization: true },
```

Comment:
I considered it as well, but for now, I think it's too much. Since the app heavily relies on manual initialization, it seems better to hardcode it here. For both cases, since they shouldn't be changed, hardcoding seems like the best approach IMO. What do you think?

Reply:
I like the idea of it being centralised, e.g. if we hardcoded these values in our `configuration` file but I am equally impartial. If it makes the most sense to have them here, we can keep it as is.